cornell-dti/cu-apts
# Homepage listings page

## Summary

- Closes issue #14 
- This pull request is for the basic implementation of the new homepage design that includes apartment listings.
### Before/After

- The background of the page is now a shade of white instead of an image.
- The search bar has been modified to match the new design requirements.
- A "Browse Renting Companies" section has been added, which can be populated with as many listings as required, and it will scroll horizontally through the listing cards so that 3 are showing at a time.
- Currently, the listing cards are populated with hardcoded data, but they have been made functional in a way that populating them with backend data will not be difficult.
### Testing

- Run `yarn install` in `frontend` to make sure all packages are installed.
- Run `yarn start` in `frontend` to view the React project in a local browser. It will open to the Home page, where you can see the implementation of the new design.
